etc/kayobe/kolla/config/fluentd/output/01-es.conf

```diff
@@ -0,0 +1,2 @@
+# Override Kolla configuration for logging directly to
+# Elasticsearch.
```

Comment:
Wasn't the fix we discussed for this to enforce predication on kolla_enable_central_logging in Kolla-Ansible?  I guess that'll take a while to work its way through upstream but it might be better to cherry-pick a fix than to overwrite the file like this.

Reply:
That's right - however as far as i'm aware there is no fix to cherry-pick yet. The bug is tracked here: https://bugs.launchpad.net/kolla-ansible/+bug/1725266
If we don't override this, it prevents fluentd sending OpenStack service logs to Monasca. As soon as we have a fix I can revert this change - does that sound OK?

Reply:
Love it - "As a user of Monasca..." - you can take the man out of Cray, but not Cray out of the man...

Reply:
Once a Crayon..